webpack.config.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/* eslint-env node */
+
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+const TerserWebpackPlugin = require( 'terser-webpack-plugin' );
+
+module.exports = {
+	mode: 'production',
+	devtool: 'source-map',
+
+	performance: {
+		hints: false
+	},
+
+	entry: path.join( __dirname, 'src', 'index.js' ),
+
+	output: {
+		library: 'CKEditor',
+
+		path: path.join( __dirname, 'dist' ),
+		filename: 'ckeditor.js',
+		libraryTarget: 'umd',
+		libraryExport: 'default'
+	},
+
+	optimization: {
+		minimizer: [
+			new TerserWebpackPlugin( {
+				sourceMap: true,
+				terserOptions: {
+					output: {
+						// Preserve license comments.
+						comments: /^!/
+					}
+				}
+			} )
+		]
+	},
+
+	plugins: [
+		new webpack.BannerPlugin( {
+			banner: `/*!*
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */`,
+			raw: true
+		} ),
+	],
+
+	module: {
+		rules: [
+			{
+				test: /\.js$/,
+				loader: 'babel-loader',
```

Comment:
We should probably specify which preset should be used and which browsers we target.

Reply:
Could you give a hand with that?

Reply:
We're already doing it in React, you can try if settings provided there works: https://github.com/ckeditor/ckeditor4-react/blob/9e5955f1c81c8670e460e2cf69352eae6e324fad/webpack.config.js#L78-L97